src/pages/receiving/steps/ReceivingStep.ts

```diff
@@ -58,11 +58,31 @@ class ReceivingStep extends BasePageModel {
     return this.page.getByRole('button').getByText('Export template');
   }
 
+  get importTemplateButton() {
+    return this.page
+      .locator('[class="btn btn-outline-secondary btn-xs mr-3"]')
```

Comment:
Can we get a better locator here?

Reply:
I think that now, we can't as this button differs from others and doesn't have role button:
![Screenshot from 2025-04-11 11-01-30](https://github.com/user-attachments/assets/e48cd3a4-d7c0-4a66-97c9-3105f83b5028)